templates/stripes/stripe-menu.php

```diff
@@ -0,0 +1,19 @@
+<?php $menu = !empty($menu_id) ? $this->getMenuLinks($menu_id) : [] ?>
+
+<?php if (!empty($menu)) : ?>
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <div class="vssl-stripe-column">
+    <?php if (count($menu['links'])) : ?>
+    <ul>
+      <?php foreach($menu['links'] as $key => $link) : ?>
```

Comment:
You may keep but PHP does not require the `$key` here if you're not going to use it.